.github/workflows/algolia-integrity.yml

```diff
@@ -11,27 +11,30 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+
       - name: Fetch
         run: git fetch origin v3
+
       - name: Setup env
         uses: the-guild-org/shared-config/setup@main
         with:
           nodeVersion: 18
+          packageManager: pnpm
 
       - name: Build Packages
-        run: yarn build
+        run: pnpm build
 
       - name: Typecheck
-        run: yarn workspace website build
+        run: pnpm --filter website build
```

Comment:
@ardatan you can drop the algolia workflows, I will add the new one later (which are `pnpm` compliant)